psga/config/pipeline-schema.yaml

```diff
@@ -0,0 +1,95 @@
+pipeline:
+  name: sars-cov-2
+  docker_image: ''
+  protocols:
+  - name: illumina-sars-cov-2-artic-v3
+    kit:
+      name: Artic V3
+      slug: articv3
+    sequencer_technology:
+      name: Illumina
+      slug: illumina
+    pathogen:
+      name: Sars-CoV-2
+      slug: sars-cov-2
+  - name: ont-sars-cov-2-artic-v4
+    kit:
+      name: Artic V4
+      slug: articv4
+    sequencer_technology:
```

Comment:
This parameter is called `sequencing_technology` in the pipeline, in agreement to the architecture documents.
Values are (case sensitive: `illumina`, `ont`, `unknown`).